browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4066,6 +4066,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_SHOW_UNDO_WHEN_TABS_CLOSED" desc="Menu item to show undo button when tabs are closed">
       Show undo button when tabs are closed
     </message>
+    <message name="IDS_BRAVE_EXIT_CONFIRMATION" desc="A confirmation shown on Exit Brave">
+      Are you sure you want to exit Brave?
```

Comment:
@rmcfadden3 @timchilds could you please confirm that it's good with such wording?

Reply:
Seems okay to me. Maybe change the dialog header to `Exit Brave?` instead.

QQ: Is there a difference between "exit" and "close" in this case? I'm wondering if `Close Brave? | Are you sure you want to close Brave?` would be a tad more user friendly. But if `Exit` is the correct word in Android world, I can be okay with that too.

Reply:
Usually on Android the confirmation dialogs titles duplicate the menu titles, like `Exit` in our case. I can definitely make it `Exit Brave`, but it's not a common practice we have with other confirmation dialogs.
@rmcfadden3 we have a menu item that is called `Exit` for a long time. I would think users expect it's Exit versus Close.
![Screenshot_20241022_132300](https://github.com/user-attachments/assets/7d7ef00e-9093-4128-966e-d3bbc2886e34)